theseus/geometry/functional/constants.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
This file is copied from https://github.com/facebookresearch/theseus/blob/main/theseus/constants.py

Reply:
I think we will use all of them very shortly. If everything goes well, `SO3` and `SE3` should be done very soon.